main/acle.md

```diff
@@ -724,6 +724,68 @@ uses the architecture. These macros are detailed in
 [Feature test macros](#feature-test-macros). All ACLE predefined macros
 start with the prefix `__ARM`.
 
+## Keyword attributes
+
+ACLE adds several non-standard keywords to C and C++. These keywords
+all start with the prefix `__arm_`. (However, not everything that
+starts with `__arm_` is a keyword.)
+
+The keywords are modeled after standard `[[…]]` attributes and
+fulfill a similar role to them.  However, the C++ standard says that
+“any [non-standard attribute] that is not recognized by the
+implementation is ignored” `[dcl.attr.grammar]`. It is therefore not
+appropriate to use standard attributes for properties that affect the
+type system, that affect the binary interface of the generated code, or
+that must be honored for other reasons. ACLE uses keywords for some of
+these properties instead. The keywords are therefore known as
+“keyword attributes”.
+
+A keyword attribute can appear wherever a standard `[[…]]` attribute
+can appear. Following the terminology of the C++ standard, a keyword
+attribute “appertains” to whatever a standard attribute would appertain
```

Comment:
“appertains” is the term that is being borrowed/quoted from the C++ standard.  I was worried that just:
> A keyword attribute appertains to whatever a standard attribute would appertain to.
wouldn't mean anything to anyone who hadn't read this part of the standard, since “appertain” isn't widely used elsewhere.  Most people would probably just say “apply”, or something like that.

And:
> Following the terminology of the C++ standard, a keyword attribute appertains to whatever a standard attribute would appertain to.
IMO is harder to parse.  Guess that's personal, though, and it sounds like it's the opposite for others.

Reply:
Thanks for explaining. I'm happy keeping the quotes, I was mostly just curious why you added them in the first place.